### PR TITLE
chore(Storybook): refactor ProgressBar, StackingGroup, Skeletons, Spinner and Titles

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-819)

### CODE REVIEW

File: src/__screenshot_tests__/__image_snapshots__/title-screenshot-test-tsx-title-1-with-link-and-long-text-desktop-1-snap.png
Comment (pladaria) [inline src/__screenshot_tests__/__image_snapshots__/title-screenshot-test-tsx-title-1-with-link-and-long-text-desktop-1-snap.png:1]: Please take a screenshot of the component only, not the entire page.

**Review by pladaria — APPROVED** [review]

File: src/__screenshot_tests__/__image_snapshots__/title-screenshot-test-tsx-title-1-with-link-and-long-text-desktop-1-snap.png
Comment (marcoskolodny) [reply src/__screenshot_tests__/__image_snapshots__/title-screenshot-test-tsx-title-1-with-link-and-long-text-desktop-1-snap.png:1]: fixed

**Review by atabel — APPROVED** [review]

File: src/__stories__/stacking-group-story.tsx
Comment (yceballost) [inline src/__stories__/stacking-group-story.tsx:1]: can you remove <ResponsiveLayout> in this story avoid margins when you increase the canvas width?

File: src/__stories__/title-story.tsx
Comment (yceballost) [inline src/__stories__/title-story.tsx:1]: I would add the possibility to show title with an icon in the right side, like:
https://tinyurl.com/2x3z622r

**Review by yceballost — APPROVED** [review]

File: src/__stories__/title-story.tsx
Comment (marcoskolodny) [reply src/__stories__/title-story.tsx:1]: added

**Pull request merged**